tests/metrics/run_docker_metrics

```diff
@@ -1,24 +1,19 @@
 #!/bin/bash
 
 source ./run_docker_metrics.dat
+
+# Verify that test-common.bash is generated.
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+[ -f "${SCRIPT_PATH}/../lib/test-common.bash" ] || \
```

Comment:
Yes, I missed `( )` to group the echo and the exit command, sorry about that, fixing...